test/resources/v5/order-recipients.test.js

```diff
@@ -0,0 +1,77 @@
+const faker = require('faker');
+
+const orderRecipientFactory = require('../../../lib/resources/order-recipients');
+const RO = require('../../..');
+const api = require('../../../lib/api');
+
+jest.mock('../../../lib/api');
+
+const mockCallBack = jest.fn();
+const mockPiiServerUrl = faker.internet.url();
+const mockProgramCode = faker.random.number();
+
+describe('v5 order-recipients', () => {
+  let orderRecipient;
+
+  beforeEach(() => {
+    orderRecipient = orderRecipientFactory('programs', mockProgramCode);
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('throws an error if not provided a `programs` context', () => {
+    expect(() => {
+      orderRecipientFactory('something else', mockProgramCode);
+    }).toThrow(Error('Can only create an order recipient object for programs'));
+  });
+
+  describe('when default (not configured)', () => {
+    describe('#storeOrderRecipient', () => {
+      it('should respond with an error if there is a bad config', async () => {
+        const requestBody = {};
+
+        await expect(orderRecipient.store(requestBody, mockCallBack)).rejects.toThrowError(
+          'piiServerUrl is not configured'
+        );
+      });
+    });
+  });
+
+  describe('when configured', () => {
+    beforeEach(() => {
+      RO.config.set('piiServerUrl', mockPiiServerUrl);
+    });
+
+    afterEach(() => {
+      RO.config.reset();
+    });
+
+    describe('#storeOrderRecipient', () => {
+      it('should contain the piiServerUrl in the request URL', async () => {
+        const member = { id: faker.random.number(), accept_language: 'en-CA' };
+
+        await orderRecipient.store({ member }, mockCallBack);
+
+        expect(api.post).toHaveBeenNthCalledWith(
+          1,
+          {
+            path: `/${mockPiiServerUrl}/api/v5/programs/${mockProgramCode}/order_recipients`,
+            member,
+          },
+          expect.any(Function)
+        );
+      });
+
+      it('should respond with an error if the params are invalid', async () => {
+        const requestBody = {};
+
+        await orderRecipient.store(requestBody, mockCallBack);
+
+        expect(mockCallBack).toHaveBeenCalledWith(expect.any(Array));
+        expect(api.post).not.toHaveBeenCalled();
+      });
+    });
+  });
```

Comment:
Question: We're only publicly exposing the `create` method from this module in the `context` module, correct? If that's untrue, we may do well to add tests for that method here.

In fact, we should add a test to ensure that the `create` method is doing what we intend when the factory is invoked. This can be part of the follow-up ticket, I suppose. We'll want to test the unhappy path, for example (i.e. the `catch`)

Reply:
I just updated the tests, they are all passing now but more extensive testing is required. 

Would it suffice to create a new ticket to cover all the things we need to do? And reference that ticket in the source code?